fossunited/fossunited/doctype/foss_event_cfp_submission/templates/foss_event_cfp_submission.html

```diff
@@ -0,0 +1,447 @@
+                </div>
+            </div>
+    
+            <div class="cfp-statistic-container" style="align-items: flex-start; padding: 16px 24px;">
+                <div class="text-xl semi-bold">
+                    {{ _("Approvability") }}
+                </div>
+                <div class="text-xl">
+                    {{ statistics["approvability"] }} %
+                </div>
+            </div>
+        </div>
+    
+        
+        {% if (frappe.session.user in get_reviewers(doc.event)["Emails"]) and not user_already_reviewed(doc.name) %}
+        <div class="form-divider mb-4"></div>
+        <div class="d-flex flex-column">
+            <div class="text-base semi-bold">
+                {{ _("Post Review Anonymously") }}
+            </div>
+            <div class=" w-100 h-25">
+                <div class="py-2 medium w-25">
+                    <label for="can-approve">Can Approve?</label>
+                    <select class="input-with-feedback form-control ellipsis bold" maxlength="14" name="can-approve" id="can-approve">
+                        <option class="my-1" value="Yes">Yes</option>
+                        <option class="my-1" value="No">No</option>
+                        <option class="my-1" value="Maybe">Maybe</option>
+                    </select>
+                </div>
+                <textarea name="review-remark" id="cfp-review-remark" class="form-control text-sm rounded-input my-3" placeholder="Write Remarks..." rows="4" cols="10"></textarea>
+                <div class="btn primary-button mb-1" onClick="post_review()">
+                    Post Review
+                </div>
+            </div>
+        </div>
+        {% endif %}
+    
+        <div class="form-divider my-4"></div>
+    
+        <div class="my-4">
+            <div class="text-xl my-4 semi-bold">
+                {{ _("Reviews") }}
+            </div>
+            {% if not doc.reviews | length > 0 %}
+                <div class="w-100 justify-content-center align-items-center p-5">
+                    <div class="text-base medium text-center ">
+                        No reviews yet.
+                    </div>
+                </div>
+            {% else %}
+                {% for review in doc.reviews %}
+                    <div class="d-flex flex-column my-4">
+                        <div class="text-base semi-bold mb-1">
+                            {{ review.reviewer }}
+                        </div>
+                        <div class="text-base">
+                            {{ review.remarks }}
+                        </div>
+                        <div class="my-1 text-sm medium">
+                            {{ format_date_time(review.creation, "d MMM, h:mm a") }}
+                        </div>
+                    </div>
+                {% endfor %}
+            {% endif %}
+        </div>
+    
+    </div>
+</div>
+
+{% endif %}
+
+
+{% endblock %}
+
+{% macro BreadCrumb(route) %}
```

Comment:
Sticking to snake case is better I think:

```suggestion
{% macro breadcrumbs(route) %}
```

Plural because generally at least 2.